assets/js/app.js

```diff
@@ -24,7 +24,7 @@ import topbar from "../vendor/topbar"
 
 let csrfToken = document.querySelector("meta[name='csrf-token']").getAttribute("content")
 let liveSocket = new LiveSocket("/live", Socket, {
-  longPollFallbackMs: null,
+  longPollFallbackMs: 2500,
```

Comment:
you may want to put this back to `null` as the fallback I believe is set in the cookie or in the localstorage and it is a PITA to go back to websockets

Reply:
I'll change that back.